modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -220,10 +220,10 @@ language" override this setting after they have logged in.
 ....
 
 === Set the default app to open on login. Use the app names as they appear in the
-URL after clicking them in the Apps menu, such as documents, calendar, and
-gallery. You can use a comma-separated list of app names, so if the first
-app is not enabled for a user then ownCloud will try the second one, and so
-on. If no enabled apps are found it defaults to the Files app.
+
+URL after clicking them in the Apps menu, such as documents and calendar. 
```

Comment:
you have to do any change for config_sample_php_parameters.adoc in core config_sample.php
this gets overwritten by the next run of the script...